src/pages/Login.tsx

```diff
@@ -23,38 +24,48 @@ const styles = {
 function Login() {
     const [showQR, setShowQR] = useState<string>();
     async function sendRequestToMobile(jwtRequests: string[], channel = 'mobile') {
+        const requests = JSON.stringify(jwtRequests);
         if (isMobile()) {
-            const requests = JSON.stringify(jwtRequests);
             window.location.replace(`${settings.config.tonomyIdLink}?requests=${requests}`);
 
             // TODO
             // wait 1-2 seconds
             // if this code runs then the link didnt work
+            setTimeout(() => {
+                alert("link didn't work");
+            }, 1000);
         } else {
-            setShowQR(JSON.stringify({ text: 'hello-world' }));
-            // Use communication microservice to send request to mobile
-            // alert('Run on browser to test');
+            const randomSeed = randomString(100);
```

Comment:
channel name should be created server side, otherwise possible to highjack another channel if you know the name